spec/models/chatroom_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe Chatroom, type: :model do
+  describe "Associations" do
+    it { should have_many(:chatroom_users) }
+    it { should have_many(:users).through(:chatroom_users) }
+    it { should have_many(:messages) }
+  end
+
+  describe "Validations" do
+    subject { FactoryBot.create(:chatroom) }
+
+    it { should validate_presence_of(:name) }
+    it { should validate_uniqueness_of(:name) }
+    it { should validate_length_of(:name), maximum: 50 }
+
+    it "is a valid within valid attributes" do
+      expect(subject).to be_valid
+    end
+
+    it "is not a valid within maximum length name more than 50" do
```

Comment:
it { should validate_length_of(:name), maximum: 50 }, đã viết cái này thì cái group này không cần biết nữa đâu